provider/blockdev_commit_base.py

```diff
@@ -94,6 +94,15 @@ def get_linux_disk_path(session, disk_size):
                 return kname
         return None
 
+    def configure_disk(self, tag):
+        if len(self.params["images"].split()) == 1:
```

Comment:
what about 'device_tag = image1 data', in this situation, configure_data_disk will be called, but for image1, which is the system image, configure_system_disk should be called.
If there is only one system image in your tests, we can simply check if tag is (self.params['images'].split())[0], if not, i.e. there are multiple system images, we have to find out the list of all images and check if tag is in the list

Reply:
OK, repace it by:
   if tag == "image1":